subsys/net/l2/wifi/test_certs/README.txt

```diff
@@ -0,0 +1,14 @@
+****************************************************************
+1. Convert CA format from PEM to DER.
+openssl x509 -outform der -in cas.pem -out ca.der
+openssl x509 -outform der -in wifiuser.pem -out client.der
+
+2. Convert RSA Key format to DER.
+openssl rsa -in wifiuser.key -outform DER -out client_key.der
+
+3. Convert .der to header file:
+xxd -i ca.der > ca.h
+
+4. Replace corresponding cert header files at below path:
+subsys/net/l2/wifi/test_certs/
+#please make sure the parameter names are same.
```

Comment:
We should move this to `doc/connectivity/networking/api/wifi.rst` under a sub-section for `Wi-Fi shell enterprise testing: Certification provisioning`

```suggestion
.. _wifi_shell_test_certs:

X.509 Certificate header generation
###################################

Wi-Fi enterprise security requires use of X.509 certificates, a test certificates
in PEM format are committed to the repo and the below steps to be followed to convert
them to a C header file that can be included in Wi-Fi shell module.

Convert CA format from PEM to DER
=================================

To convert the CA format from PEM to DER, use the following command:

.. code-block:: none

    openssl x509 -outform der -in cas.pem -out ca.der

Convert RSA Key format to DER
=============================

To convert the RSA Key format to DER, use the following command:

.. code-block:: none

    openssl rsa -in wifiuser.key -outform DER -out client_key.der

Convert .der to header file
===========================

To convert the .der file to a header file, use the following command:

.. code-block:: none

    xxd -i ca.der > ca.h

Replace corresponding cert header files
=======================================

Replace the corresponding cert header files at the following path:

```
subsys/net/l2/wifi/test_certs/
```

.. note:: Please ensure that the parameter names are the same.
```

Reply:
Hi @krish2718 , updated, please review again